### PR TITLE
aws-s3-presign: add page

### PR DESCRIPTION
**Pull request by shermanhui**
PR to address #8406

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

Screen Shot 2022-10-20 at 19 36 34

### CODE REVIEW

File: pages/common/aws-s3-presign.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# aws s3 presign
+
+> Generate a pre-signed URL for an Amazon S3 object.
```
Comment (navarroaxel) [inline pages/common/aws-s3-presign.md:3]:
```suggestion
> Generate pre-signed URLs for Amazon S3 objects.
```

File: pages/common/aws-s3-presign.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/presign.html>.
+
+- Generate a pre-signed URL for a specific bucket and key that is valid for one hour:
+
+`aws s3 presign s3://{{example_bucket_name}}/{{file_name}}.txt`
```
Comment (navarroaxel) [inline pages/common/aws-s3-presign.md:8]:
```suggestion
`aws s3 presign s3://{{bucket_name}}/{{path/to/file}}`
```

File: pages/common/aws-s3-presign.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# aws s3 presign
+
+> Generate a pre-signed URL for an Amazon S3 object.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/presign.html>.
+
+- Generate a pre-signed URL for a specific bucket and key that is valid for one hour:
```
Comment (navarroaxel) [inline pages/common/aws-s3-presign.md:6]:
```suggestion
- Generate a pre-signed URL for a specific S3 object that is valid for one hour:
```

File: pages/common/aws-s3-presign.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+`aws s3 presign s3://{{example_bucket_name}}/{{file_name}}.txt`
+
+- Generate a pre-signed URL with a custom lifetime:
+
+`aws s3 presign s3://{{example_bucket_name}}/{{file_name}}.txt --expires-in {{604800}}`
```
Comment (navarroaxel) [inline pages/common/aws-s3-presign.md:12]:
```suggestion
`aws s3 presign s3://{{bucket_name}}/{{path/to/file}} --expires-in {{duration_in_seconds}}`
```

File: pages/common/aws-s3-presign.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# aws s3 presign
+
+> Generate a pre-signed URL for an Amazon S3 object.
+> More information: <https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3/presign.html>.
+
+- Generate a pre-signed URL for a specific bucket and key that is valid for one hour:
+
+`aws s3 presign s3://{{example_bucket_name}}/{{file_name}}.txt`
+
+- Generate a pre-signed URL with a custom lifetime:
```
Comment (navarroaxel) [inline pages/common/aws-s3-presign.md:10]:
```suggestion
- Generate a pre-signed URL valid for a specific lifetime:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**